components/application/widgets/Scheduler.js

```diff
@@ -451,6 +461,46 @@ export function formatDate(inDate) {
   return dateJson;
 }
 
+function convertPosition(position) {
```

Comment:
Some other components may use this function, such as priority selector, and tag selector. This function can be put in a public library.

Reply:
Ok, It has been placed in a new file